parquet-column/src/main/java/org/apache/parquet/column/statistics/Statistics.java

```diff
@@ -35,6 +35,44 @@
  */
 public abstract class Statistics<T extends Comparable<T>> {
 
+  /**
+   * Builder class to build Statistics objects. Used to read the statistics from the Parquet file.
+   */
+  public static class StatisticsBuilder {
```

Comment:
Why add this to the public API?

Because this is nested, it should be `Statistics.Builder`, not `Statistics.StatisticsBuilder`.

Reply:
It needs to be public because `ParquetMetadataConverter` uses it. The concept is that you are only able to initialize `num_nulls` to `-1` (meaning it is not set in the file) by using the builder (read path). If you are creating a new `Statistics` object in any other way (write path) `num_nulls` will be initialized to `0`.
I'll rename it to Builder.